lightning-tag/basic-implementation.md

```diff
@@ -0,0 +1,54 @@
+# Base Lightning Tag Implementation 
+
+!> This document provides information on how to implement Lightning Tag in its basic form. Throughout this page, there will be links to documentation detailing additional features and use-cases that can be deployed
+
+The base version of Lightning Tag will only:
+
+1. Execute data collection rules that you have configured in cooperation with your Lotame account representative
+1. Fire subscribed sync pixels
+
+This version of the tag is right for you if:
```

Comment:
I think we can remove this section entirely and rely on the fact that is named Basic Implementation and there are specific links at the bottom for adding these functions. I would rather people start with this (incorrectly) and have to add to it than start with one of the more complicated (incorrectly) and be making a bunch of unnecessary work for turnstyle.